test/integration/steps/test_parent_steps.py

```diff
@@ -552,3 +620,43 @@ def sequence_id_error_logs(context):
     )
     logs = get_logs(query, "processor", context["start_time"])
     assert logs != [], "ERROR!!.. Sequence id error message not found."
+
+
+@then("an invalid opening times error is generated")
+def invalid_opening_times_error(context):
+    query = (
+        f'fields message | sort @timestamp asc | filter correlation_id="{context["correlation_id"]}"'
+        ' | filter report_key like "INVALID_OPEN_TIMES"'
+    )
+    logs = get_logs(query, "processor", context["start_time"])
+    assert "misformatted or illogical set of opening times." in logs, "ERROR!!.. error message not found."
+
+
+@then("the opening times changes are marked as valid")
+def no_opening_times_errors(context):
+    query = (
+        f'fields message | sort @timestamp asc | filter correlation_id="{context["correlation_id"]}"'
+        ' | filter message like "Specified opening times not equal"'
+    )
+    logs = get_logs(query, "processor", context["start_time"])
+    assert logs != [], "ERROR!!.. log messages showing in cloudwatch."
+
+
```

Comment:
In `test/integration/steps/pytest.ini` can you add these lines under the pytest title
```
filterwarnings =
    ignore::DeprecationWarning
```

Reply:
This comment is unrelated to this file. But the deprecation warning is due to this `DeprecationWarning: invalid escape sequence \/`